executor/builder.go

```diff
@@ -5226,16 +5226,24 @@ func (b *executorBuilder) buildBatchPointGet(plan *plannercore.BatchPointGetPlan
 		// `SELECT a FROM t WHERE a IN (1, 1, 2, 1, 2)` should not return duplicated rows
 		handles := make([]kv.Handle, 0, len(plan.Handles))
 		dedup := kv.NewHandleMap()
+		// Used for clear paritionIDs of duplicated rows.
+		dupPartPos := 0
 		if plan.IndexInfo == nil {
-			for _, handle := range plan.Handles {
+			dupPartID := len(plan.PartitionIDs) >= len(plan.Handles)
```

Comment:
Why need check `>=` here?

Reply:
Maybe check `len(plan.PartitionIDs) == 0` is enough.

Reply:
> Why need check `>=` here?

Just ignore it，because i'm not sure if "len(plan.PartitionIDs) > 0" is enough.